website/docs/prysm-usage/checkpoint-sync.md

```diff
@@ -4,68 +4,207 @@ title: Sync from a checkpoint
---checkpoint-state=$PWD/state_prater_altair_3041920-0x34ebc10f191706afbbccb0c3c39679632feef0453fe842bda264e432e9e31011.ssz \
+--checkpoint-block=$PWD/block_goerli_bellatrix_3041920-0x766bdce4c70b6ee991bd68f8065d73e3990895b1953f6b931baae0502d8cbfcf.ssz \
+--checkpoint-state=$PWD/state_goerli_bellatrix_3041920-0x34ebc10f191706afbbccb0c3c39679632feef0453fe842bda264e432e9e31011.ssz \
 --genesis-state=$PWD/genesis.ssz
 ```
 
+  </TabItem>
+</Tabs>
+
+
+
+## Verify the authenticity of your beacon node's checkpoint
+
+To verify that the checkpoint state you're using is legitimate, follow these steps after starting your beacon node with checkpoint sync enabled:
+
+1. Navigate to `http://localhost:3500/eth/v1/beacon/headers/finalized` using your browser.
+2. Find the `slot` number and `state_root` value.
+3. Use a trusted blockchain explorer to verify the `state_root`. Navigate to one of the following pages, replacing `SLOT` with the `slot` you pulled from your browser:
+   - Prater: https://prater.beaconcha.in/block/SLOT
+   - Sepolia: https://sepolia.beaconcha.in/block/SLOT
+   - Ropsten: https://ropsten.beaconcha.in/block/SLOT
+   - Mainnet: https://beaconcha.in/block/SLOT
+4. Ensure that the `state_root` reported by the blockchain explorer matches the `state_root` you pulled from your browser. If you don't see a match, feel free to reach out to us on [Discord](https://discord.gg/prysmaticlabs) and we'll help you troubleshoot.
+
+
+## Frequently asked questions
+
+**Is checkpoint sync less secure than syncing from genesis?** <br/>
+No. It's actually considered *more* secure thanks to the protections against long-range attacks afforded by [Weak Subjectivity](https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/).
+
+**How and when are "finalized checkpoints" created?** <br/>
+TODO
+
+**Can I use checkpoint sync on any network?** <br/>
+TODO
+
+**Can I use checkpoint sync with an existing, partially synced database?** <br/>
+No - checkpoint sync requires a fresh, unused data directory.
+
+**Are there any publicly available, trustworthy checkpoint sync endpoints that I can use?** <br/>
+The Ethereum Foundation DevOps team runs a handful of checkpoint sync endpoints that can be used for testnets:
+
+ - Goerli/Prater: https://goerli.checkpoint-sync.ethdevops.io
+ - Ropsten: https://ropsten.checkpoint-sync.ethdevops.io
+ - Sepolia: https://sepolia.checkpoint-sync.ethdevops.io
+
+
+**Does the Prysm team host checkpoint sync files that I can use?** <br/>
+TODO
+
+**Do I need to provide a genesis state when using checkpoint sync on Mainnet?** <br/>
+TODO
+
+**Will I be able to use Infura as a checkpoint state provider after The Merge?** <br/>
+TODO
+
+**I started a beacon node with checkpoint sync enabled. Can I have other nodes pull checkpoint state from this node right away, or do I need to wait for it to become fully synced?** <br/>
+TODO
```

Comment:
TODO

Reply:
This hasn't been tested, but the block and state are immediately finalized, so the RPC server should be able to serve them to other clients.